scala-generator/src/main/scala/models/Play2JsonStandalone.scala

```diff
@@ -43,23 +43,32 @@ package ${ssd.namespaces.models} {
     import play.api.libs.json.JsString
     import play.api.libs.json.Writes
     import play.api.libs.functional.syntax._
+    import org.joda.time.format.DateTimeFormat
+    import org.joda.time.format.DateTimeFormatterBuilder
 ${JsonImports(form.service).mkString("\n").indent(4)}
 
+    private[${ssd.namespaces.last}] val DateTimeFormatter =
+      new DateTimeFormatterBuilder().append(
+        org.joda.time.format.ISODateTimeFormat.dateTime.getPrinter,
```

Comment:
`DateTimeFormat` and `DateTimeFormatBuilder` are imported, maybe import `org.joda.time.format.ISODateTimeFormat` as well?